src/JustSaying/Naming/NamingConventionExtensions.cs

```diff
@@ -0,0 +1,25 @@
+using System;
+
+namespace JustSaying.Naming
+{
+    public static class NamingConventionExtensions
```

Comment:
`///` docs for all members and the type.

Reply:
Done!